src/ResourceManager/Scheduler/Commands.Scheduler.Test/ScenarioTests/AzureRM.Resources.ps1

```diff
@@ -0,0 +1,151 @@
+﻿function Get-AzureRmResourceGroup
```

Comment:
I removed AzureRm.Resources.ps1 from the project with the second commit. Not sure how it got there in first place. Thanks.